backend/src/main/java/com/sep6/backend/security/config/SecurityConfiguration.java

```diff
@@ -43,7 +43,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .authorizeHttpRequests()
                 .requestMatchers(
                         "/accounts/**",
-                        "/movies/*/reviews/")
+                        "/movies/*/reviews")
```

Comment:
What about /movie/{movieId}/reviews/{reviewId}? I believe  we will need to include also another request matcher for /movies/*/reviews/*

Reply:
okay, then it should be /movies/*/reviews/**

Reply:
Done